src/ngraph/op/convolution.hpp

```diff
@@ -377,6 +320,6 @@ namespace ngraph
                                                  size_t output_channel_axis_filters,
                                                  size_t batch_axis_result,
                                                  size_t output_channel_axis_result);
-        }
-    }
-}
+        } // namespace util
+    }     // namespace op
```

Comment:
The indenting of the namespace comment is inconsistent on all files which have it. I would just remove these comments. We don't use them elsewhere in ngraph core

Reply:
clang-format seems to be inserting those comments. I remove them and reformat and they come back.

Reply:
Or it's VSC adding them; I am removing them but they are likely to return in the future.